### PR TITLE
Updated the response for api endpoints & added additional info

### PR DESCRIPTION
**Pull request by ajithkumar-maragathavel**
## Purpose of this pull request

This pull request (PR) updates the response from the API endpoints and also added additional information on the visibility attribute.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/rest/tutorials/configurable-product/create-configurable-product.html
- https://devdocs.magento.com/guides/v2.3/rest/tutorials/configurable-product/plan-product.html

### CODE REVIEW

File: src/guides/v2.3/rest/tutorials/configurable-product/create-configurable-product.md [hunk 1]
```diff
@@ -19,6 +19,15 @@ The sample payload does not contain the price or the size. These attributes will
...
 The `visibility` attribute is set to 4, meaning the product can be found by browsing or searching. This value will be changed for the simple products.
 
+The below table shows the equivalent code for the visibility attribute
+
+Visibility | Code
+--- | ---
+`Not Visible Individually` | 1
+`Catalog` | 2
+`Search` | 3
+`Catalog,Search` | 4
```
Comment (BarnyShergold) [inline src/guides/v2.3/rest/tutorials/configurable-product/create-configurable-product.md:29]:
```suggestion
`Catalog, Search` | 4
```

File: src/guides/v2.3/rest/tutorials/configurable-product/plan-product.md [hunk 2]
```diff
@@ -53,15 +53,19 @@ We'll make several calls to find the values needed to create the product
 
 The sample data provides multiple attribute sets, including Default, Top, and Bottom. To assign the Top attribute set to the product, we need to know the corresponding `attribute_set_id`.
 
-Use the following call to search for the attribute set named `Top`.
+Use the following call to search for the attribute set named `Top` and entity_type_id equals to 4.
```
Comment (BarnyShergold) [inline src/guides/v2.3/rest/tutorials/configurable-product/plan-product.md:56]:
```suggestion
Use the following call to search for the attribute set named `Top` and `entity_type_id` equals to `4`.
```

File: src/guides/v2.3/rest/tutorials/configurable-product/plan-product.md [hunk 3]
```diff
@@ -53,15 +53,19 @@ We'll make several calls to find the values needed to create the product
...
+By default, the `catalog_product` entity has the entity_type_id value as 4.
```
Comment (BarnyShergold) [inline src/guides/v2.3/rest/tutorials/configurable-product/plan-product.md:57]:
```suggestion
By default, the `catalog_product` entity has the `entity_type_id` value as `4`.
```

File: src/guides/v2.3/rest/tutorials/configurable-product/plan-product.md [hunk 4]
```diff
@@ -101,11 +114,51 @@ Use the `GET V1/products/attribute-sets/:attributeSetId/attributes` call to retu
...
+        "backend_type": "int",
+        "source_model": "Magento\\Eav\\Model\\Entity\\Attribute\\Source\\Table",
+        "default_value": "",
+        "is_unique": "0",
+        "validation_rules": []
+    }
+]
+```
+
+The response contains almost 3,000 lines and above is the little bit of response from the endpoint. The following table provides a summary of the attributes that are relevant in this tutorial.
```
Comment (BarnyShergold) [inline src/guides/v2.3/rest/tutorials/configurable-product/plan-product.md:161]:
```suggestion
The response contains almost 3,000 lines and the above is a part of the response from the endpoint. The following table provides a summary of the attributes that are relevant in this tutorial.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just a few formatting and syntax issues

File: src/guides/v2.3/rest/tutorials/configurable-product/plan-product.md (hunk 3 above)
Comment (ajithkumar-maragathavel) [reply src/guides/v2.3/rest/tutorials/configurable-product/plan-product.md]: @BarnyShergold Thanks for the review. I have committed the changes.

**Review by BarnyShergold — APPROVED** [review]

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**